### PR TITLE
Fix IPv6 subnet allocation.

### PR DESCRIPTION
**Pull request by mykaul**
Now it uses the same subnet as IPv4.

### CODE REVIEW

**Comment by mykaul** [comment]
ci test please

File: lago/virt.py [hunk 1]
```diff
@@ -391,7 +393,7 @@ def _libvirt_xml(self):
             )
             net_xml.append(domain_xml)
         if 'dhcp' in self._spec:
-            IPV6_PREFIX = 'fd8f:1391:3a82:5e0d::'
+            IPV6_PREFIX = 'fd8f:1391:3a82:' + subnet + '::'
```
Comment (nvgoldin) [inline lago/virt.py]: don't we need to convert it to IPV6 format?

Comment (nvgoldin) [inline lago/virt.py]: (or I missed something here)

Comment (mykaul) [inline lago/virt.py]: @nvgoldin - it translates to IPv6 eventually, no worries. It's just few octets out of them, for example:
fd8f:1391:3a82:204::/64 - for 192.168.204 network.

**Comment by mykaul** [comment]
ci merge please

**Comment by nvgoldin** [comment]
ci merge please

**Pull request merged**